src/lang-promql/complete/hybrid.ts

```diff
@@ -281,4 +253,54 @@ export class HybridComplete implements CompleteStrategy {
           );
         });
   }
+
+  private arrayToCompletionResult(
+    data: AutoCompleteNode[],
+    from: number,
+    to: number,
+    context: AutocompleteContext,
+    state: EditorState,
+    includeSnippet = false
+  ): CompletionResult {
+    const text = state.sliceDoc(from, to);
+    const options: Completion[] = [];
+    const fuzzyOption = { pre: this.fuzzyPre, post: this.fuzzyPost };
+
+    for (const completionList of data) {
+      for (const label of completionList.labels) {
+        if (context.filterType === FilterType.Fuzzy) {
+          const result = fuzzy.filter(text, [label], fuzzyOption);
```

Comment:
Somewhere, we will have to HTML-escape label names (since they can contain arbitrary strings).

Reply:
should be fine now ;)